api/endpoints/new-schedule-link.js

```diff
@@ -21,11 +21,15 @@ export default async (req, res) => {
     })
   }
 
+  // should open a meeting using
+  // <staging-app-url>/api/endpoints/schedule-link?id=9whz6
```

Comment:
Was this intentional? It looks like it's a comment about the hack-night logic (because the id `9whz6` is used for that call), but it's not near any of the hack-night logic.

Reply:
removed that id